x-pack/plugins/security_solution/public/timelines/components/fields_browser/field_table_columns/index.tsx

```diff
@@ -143,7 +141,7 @@ export const useFieldTableColumns: UseFieldTableColumns = ({
                     width={actions.length > 0 ? '335px' : '400px'}
                     data-test-subj={`field-${name}-description`}
                   >
-                    {`${description ?? getEmptyValue()} ${getExampleText(example)}`}
+                    {`${description ?? getEmptyValue()}`}
```

Comment:
What was `example` providing?

Reply:
There **should** be text showing an example of data to put in the field, however it almost never appeared due to the text being truncated, and then it was not included in the hover-over.

<img width="427" alt="Screen Shot 2023-03-27 at 12 13 23 PM" src="https://user-images.githubusercontent.com/915763/228001274-3d182f4e-da1b-4301-af4b-52f8a68fc875.png">

Reply:
Ahhh ok. Yea I couldn't find it out in the wild.

Reply:
🤷